src/components/Header.tsx

```diff
@@ -6,25 +6,32 @@ export interface HeaderProps {
   imgSrc?: string;
   alt?: string;
   logOut?: string;
+  loggedIn?: boolean;
 }
 
 export default class Header extends React.Component<HeaderProps, {}> {
+  static defaultProps = {
+    loggedIn: false,
+  };
+
   render(): JSX.Element {
     let src = this.props.imgSrc ? require(`${this.props.imgSrc}`) : "";
     return(
-      <nav className="navbar navbar-default">
+      <nav className="reusable-navbar navbar-default clearfix">
```

Comment:
I did not update this in the other app because I was going to do it here, but since you're working on it... this should be a `<header>` element. The `nav` element is only for navigation-related elements. This is tricky but I think there should be no `nav` in this component. Right now it only has a title and a logo so those are not part of it. You can argue that the logout link should be surrounded by a `nav` but it's not always there so I think that's enough to exclude the `nav` element here.